library/LuaApi.cpp

```diff
@@ -1719,6 +1719,7 @@ static const LuaWrapper::FunctionReg dfhack_units_module[] = {
     WRAPM(Units, isFortControlled),
     WRAPM(Units, isOwnCiv),
     WRAPM(Units, isOwnGroup),
+    WRAPM(Units, isNobleFromOtherSite),
```

Comment:
needs documentation in `docs/dev/Lua API.rst` and `docs/changelog.txt` entries in the API and Lua sections